apps/web/components/apps/App.tsx

```diff
@@ -127,7 +127,7 @@ const Component = ({
         <div className="mb-8 flex pt-4">
           <header>
             <div className="mb-4 flex items-center">
-              <img className="min-h-16 min-w-16 h-16 w-16" src={logo} alt={name} />
+              <img className="min-h-16 min-w-16 h-16 w-16" src={WEBAPP_URI + logo} alt={name} />
```

Comment:
Can we not remove all these usages of WEBAPP_URI by setting base href. My concern is that developers will certainly forget adding that WEBAPP_URI